pkgs/tools/text/num-utils/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "num-utils";
+  version = "0.5";
+
+  src = fetchurl {
+    url = "https://suso.suso.org/programs/num-utils/downloads/num-utils-${version}.tar.gz";
+    sha256 = "0kn6yskjww2agcqvas5l2xp55mp4njdxqkdicchlji3qzih2fn83";
+  };
+
+  buildInputs = [ perl ];
```

Comment:
If perl is run at build time we need to change that to nativeBuildInputs.

Reply:
It's not used at build time anymore, I removed that dependency

Reply:
Can this line be removed?